### PR TITLE
INF-9818 Added heatmap for express runs

### PR DESCRIPTION
**Pull request by bmukheja**

### CODE REVIEW

File: public/configurations/dashboards/infraRegressResults.json [hunk 1]
```diff
@@ -41,6 +43,31 @@
         }
     ],
     "defaultFilterOptionsOverride":{
+        "Branch": {
+            "parameter": "branch",
+            "options": [
```
Comment (ronakmshah) [inline public/configurations/dashboards/infraRegressResults.json]: Default value?

File: public/configurations/visualizations/inf-regress-express-lastrun-heatmap.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+  "id": "inf-regress-express-lastrun-heatmap",
+  "graph": "HeatmapGraph",
+  "title": "Last 20 days Express suites run results",
+  "description": "Colors of the result depict these outcomes.\n\nGray -> Skip\nRed -> Fail\nGreen -> Passed.",
+  "author": "Ronak Shah",
```
Comment (ronakmshah) [inline public/configurations/visualizations/inf-regress-express-lastrun-heatmap.json]: nit

File: src/configs/nuage/elasticsearch/tabification/index.js [hunk 3]
```diff
@@ -10,3 +10,4 @@ export { default as defaultTabify } from './defaultTabify';
 export { default as areaHeatmap } from './areaHeatmap';
 export { default as aqlCoverage } from './aqlCoverage';
 export { default as aqlPassFail } from './aqlPassFail';
+export { default as expressHeatmap } from './expressHeatmap';
```
Comment (ronakmshah) [inline src/configs/nuage/elasticsearch/tabification/index.js]: What is the delta between expressHeatmap and areaHeatmap? Do we need this much of a duplication of code?

File: public/configurations/queries/inf-regress-express-lastrun-heatmap.json [hunk 4]
```diff
@@ -0,0 +1,132 @@
+{
```
Comment (ronakmshah) [inline public/configurations/queries/inf-regress-express-lastrun-heatmap.json]: Is all the delta between this file and https://github.com/nuagenetworks/visualization-framework/blob/nuage-infra/public/configurations/queries/inf-regress-area-testsuite-heatmap.json valid?

Comment (bmukheja) [reply public/configurations/queries/inf-regress-express-lastrun-heatmap.json]: The filtered suites are different in both, so we should do it at the elasticsearch level

File: src/configs/nuage/elasticsearch/tabification/index.js (hunk 3 above)
Comment (bmukheja) [reply src/configs/nuage/elasticsearch/tabification/index.js]: This is a correct comment, in fact all the tabification files need to be trimmed down more. Can I do it via a separate PR and leave the custom function intact in all these?

File: public/configurations/dashboards/infraRegressResults.json (hunk 1 above)
Comment (bmukheja) [reply public/configurations/dashboards/infraRegressResults.json]: Added

File: src/configs/nuage/elasticsearch/tabification/index.js (hunk 3 above)
Comment (ronakmshah) [reply src/configs/nuage/elasticsearch/tabification/index.js]: Ok

**Pull request merged**